pkg/security/module/module.go

```diff
@@ -191,21 +191,6 @@ func (m *Module) Start() error {
 		}
 	}()
 
-	m.wg.Add(1)
```

Comment:
Why did you remove this ?

Reply:
@paulcacheux: I think I need your help to resolve this one as I'm not positive here.